modules/audit-logging/src/Volo.Abp.AuditLogging.Domain.Shared/Volo/Abp/AuditLogging/Localization/en.json

```diff
@@ -85,11 +85,11 @@
     "AuditLogSettingsGeneral": "General",
     "AuditLogSettingsGlobal": "Global",
     "DisplayName:IsPeriodicDeleterEnabled": "Enable clean up service system wide",
-    "Description:IsPeriodicDeleterEnabled": "If this option is disable the periodic deleter won't work. Audit logs won't be deleted automatically.",
+    "Description:IsPeriodicDeleterEnabled": "If this option is disabled the periodic deleter won't work. Audit logs won't be deleted automatically.",
     "DisplayName:GlobalIsExpiredDeleterEnabled": "Enable clean up service for all tenants and host",
-    "Description:GlobalIsExpiredDeleterEnabled": "If this option is enable all tenants and host expired items deleted automatically. Unless has a specific setting.",
+    "Description:GlobalIsExpiredDeleterEnabled": "If this option is enabled all the tenants and the host expired items will be deleted automatically. Unless has a specific setting.",
```

Comment:
If this option is enabled all the tenants and the host expired items will be deleted automatically unless it has a specific setting.